week-1-ts-ecosystem/w01e01-typescript-basics/src/ageUtils.ts

```diff
@@ -0,0 +1,5 @@
+import type {IUser} from "./types/user";
+
+export function hasGivenAge(requiredAge: number) {
+	return (user: IUser): boolean => user.age >= requiredAge
+}
```

Comment:
zamiast `utlis` nadałbym precyzyjniejszą nazwę `validators` na przykład, można by też zastanowić się czy nie powiązać to w jednym pliku jako `user.validators.ts` i tam `export` do każdej funkcji :)